sources/OpenMcdf.Extensions/OLEProperties/PropertySetStream.cs

```diff
@@ -183,7 +183,7 @@ public void Write(System.IO.BinaryWriter bw)
 
                 int size1 = (int)(bw.BaseStream.Position - oc1.OffsetPS);
 
-                bw.Seek(oc1.OffsetPS + 4, System.IO.SeekOrigin.Begin);

```

Comment:
This +4 was causing it to overwrite the property count rather than the size (I ended up with a size of 0 and >100 properties, which is just wrong)